### ISSUE TITLE
Crash with CSGCombiner

### ISSUE THREAD

**Issue by DavidGasku** [godotengine/godot#84344]
### Godot version

v4.2.beta4.official [93cdacbb0]

### System information

Manjaro linux

### Issue description

The scene use some CSG combinator, but crashes while manipulating them. Doesn't crash on beta3, so it seems to be a regression.
```
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.2.beta4.official (93cdacbb0a30f12b2f3f5e8e06b90149deeb554b)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[1] /usr/lib/libc.so.6(+0x3e710) [0x7fd5b4c31710] (??:0)
[2] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x2bc6cb4] (??:0)
[3] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x5d9915] (??:0)
[4] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x60a898] (??:0)
[5] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x3be1ad0] (??:0)
[6] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x2624f34] (??:0)
[7] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x540723] (??:0)
[8] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x3c03553] (??:0)
[9] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x3c27d3e] (??:0)
[10] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x76771c] (??:0)
[11] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x66b7d8] (??:0)
[12] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x212a712] (??:0)
[13] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x60aae2] (??:0)
[14] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x3be1ad0] (??:0)
[15] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x2121ba3] (??:0)
[16] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x213ebdf] (??:0)
[17] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x213f751] (??:0)
[18] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x514b9a] (??:0)
[19] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x450c7f] (??:0)
[20] /usr/lib/libc.so.6(+0x27cd0) [0x7fd5b4c1acd0] (??:0)
[21] /usr/lib/libc.so.6(__libc_start_main+0x8a) [0x7fd5b4c1ad8a] (??:0)
[22] /home/david/Applications/Godot_v4.2-beta4_linux.x86_64() [0x46445e] (??:0)
-- END OF BACKTRACE --
```

### Steps to reproduce

Start the project, change the exported var 'Length' in the inspector, the program will crash

### Minimal reproduction project

[CGShape Bug.zip](https://github.com/godotengine/godot/files/13235031/CGShape.Bug.zip)

**Comment by jsjtxietian** [comment]
Can be fixed by #84338

**Comment by akien-mga** [comment]
Thanks for the report! Consolidating in #84304 (same root issue).
